spec/lib/danger/ci_sources/jenkins_spec.rb

```diff
@@ -86,7 +86,7 @@
         expect(described_class.validates_as_ci?(valid_env)).to be true
       end
 
-      it "doesnt validate when require env variables are not set" do
+      it "doesn't validate when require env variables are not set" do
```

Comment:
I believe the convention is to no use contractions for rspec examples. At least that's what I've been trying to do